app-office/Focuswriter/focuswriter-1.6.7.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="A QT Word Processor"
+DESCRIPTION="Focuswriter is an Open Source QT Word Processor"
+HOMEPAGE="https://gottcode.org/focuswriter/"
```

Comment:
The '/' at the end of the URL can be removed.

Reply:
It was required before. When did it changed?

Reply:
Long time ago ...